src/Api/Managers/DocumentManager.cs

```diff
@@ -50,6 +50,9 @@ internal EntityResult UpdateDocument(int topicId, string identity, string htmlCo
                 DbContext.Add(document);
             }
 
+            // Angular app trims img-tags to non-valid xml, let us repair it
+            htmlContent = Regex.Replace(htmlContent, "<img ([^>]*)?>", "<img $1 />");
```

Comment:
yes we can.

OnlyOffice returns the images in the form `<img src="base64encodedString" />` Angular the just deletes the `/`. same happens with `<img></img>`.